src/main/java/com/hotel/api/book/filter/CustomFilter.java

```diff
@@ -22,10 +23,9 @@ public CustomFilter() {
 
     @Override
     public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain) throws IOException, ServletException {
-        String token = ((HttpServletRequest) req).getHeader("authorization");
-        MDC.put("token",((HttpServletRequest) req).getHeader("authorization").substring("Bearer ".length()));
+        MDC.put(Constants.TOKEN,((HttpServletRequest) req).getHeader("authorization").substring("Bearer ".length()));
```

Comment:
cant we change "authorization" to a contant. HttpHeader.AUTHORIZATION should work I think